editor/src/components/navigator/navigator-item/item-label.tsx

```diff
@@ -223,6 +226,8 @@ export const ItemLabel = React.memo((props: ItemLabelProps) => {
               ? colorTheme.dynamicBlue.value
               : isConditionalClause
               ? colorTheme.fg7.value
+              : isInvalidOverride
```

Comment:
I feel like this should really be a function with regular if..else statements at this point, it's 4 ternaries deep. :(